src/docs/reference/backups.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Backups
+---
+
+Backups are a feature that enables point in time data recovery for [volumes](/reference/volumes) on Railway.
+
+## How it works
+
+When mounted to a service, backups can be manually created, deleted and restored. And they can also work on a Daily/Weekly/Monthly schedule.
+
+Schedules are beta and exist behind a feature-flag attached to the service.
+
+## Retention
+
+- Daily backups are taken every day and are kept for 6 days
+- Weekly backups are taken every week and are kept for 4 weeks
+- Monthly backups are taken every month and are kept for 3 months
+
+## Pricing
+
+Backups are incremental and Copy-on-Write, we only charge for the data exclusive to them, that aren't from other snapshots or the volume itself.
+
+Backups are billed like volumes, at **$0.25 / GB**, billed monthly.
+
+## Caveats
+
+Backups are a newer feature that is still under development. Here are some limitations of which we are currently aware:
+
+- Backup incremental sizes are cached for a couple of hours when listed in the frontend, so they may show slightly stale data
+- Wiping a volume deletes all backups
+
+## Q&A
+
+- Can I download my backups?
+
+  No, if you want to download the content of the backup restore it and use something like the File Browser Template
+
+- Can we have Railway send the backups to S3?
+
+  No, if you want to send backups to S3 you should use an application specific template, like Postgres S3 Backups
+
+- Can the backups be done on a Cronjob / Schedule?
+
+  We will implement backups scheduling soon
+
+- Will Hobby users ever get the backup feature?
+
+  There are no plans yet, but we haven't discarted the possibility. Let's roll out the full backup engine for pro customers first.
```

Comment:
Feels strange, this reads as if we are talking in the 3rd party.

Reply:
Removed that entire section! It was more noisy than helpful